src/main/java/org/datavyu/plugins/ffmpeg/MpvMediaPlayer.java

```diff
@@ -1,153 +1,63 @@
 package org.datavyu.plugins.ffmpeg;
 
-//import sun.awt.windows.WComponentPeer;
-import com.sun.javafx.tk.TKStage;
-import javafx.stage.Stage;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 import org.datavyu.util.NativeLibraryLoader;
-import sun.reflect.generics.reflectiveObjects.NotImplementedException;
-
-
-import javax.swing.*;
-import java.awt.*;
-import java.lang.reflect.Method;
 import java.net.URI;
+import java.util.ArrayList;
+import java.util.List;
 
 
-public class MpvMediaPlayer extends FfmpegMediaPlayer{
+public abstract class MpvMediaPlayer extends FfmpegMediaPlayer {
+    /**
+     * Holds the library name and version
+     */
+    private final static class LibraryDependency {
```

Comment:
This class should be a util class used by all the players (Java/SDL player as well) in order to be consistent in the way we extract dependencies

Reply:
Sure. Even better: I added it to the NativeLibraryLoader.